src/Tools/ExternalAccess/Xamarin/Microsoft.CodeAnalysis.ExternalAccess.Xamarin.csproj

```
@@ -0,0 +1,36 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Library</OutputType>
+    <RootNamespace>Microsoft.CodeAnalysis.ExternalAccess.Xamarin</RootNamespace>
+    <TargetFramework>net472</TargetFramework>
+
+    <!-- NuGet -->
+    <IsPackable>true</IsPackable>
+    <PackageId>Microsoft.CodeAnalysis.ExternalAccess.TypeScript</PackageId>
```

❗️ This conflicts with an existing package

```suggestion
    <PackageId>Microsoft.CodeAnalysis.ExternalAccess.Xamarin.Remote</PackageId>
```